kubernetes/samples/charts/voyager/README.md

```diff
@@ -118,6 +118,66 @@ $ curl -k -H 'host: domain1.org' https://${HOSTNAME}:30305/testwebapp/
 $ curl -k -H 'host: domain2.org' https://${HOSTNAME}:30307/testwebapp/
 ```
 
+## SSL Termination at Ingress Controller
```

Comment:
Please make the same corrections here as the ones above for the Traefik README.

Reply:
done